### PR TITLE
Add buff and quirk icons to extended employee info

### LINKED ISSUE #50: extend `ExtendedEmployeeInfo` overlay
Display quirks and buffs in `ExtendedEmployeeInfo` overlay

### PR DESCRIPTION
**Pull request by mertwole**
Resolves #50

### CODE REVIEW

File: JamGame/Assets/Scripts/Employee/ExtendedInfo/View.cs [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        {
+            employee.AppliedBuffsChanged -= OnBuffsChanged;
+        }
+
+        private void Awake()
+        {
+            cam = Camera.main;
+
+            personality = GetComponentInParent<PersonalityImpl>();
+            employee = GetComponentInParent<EmployeeImpl>();
```
Comment (Kiuh) [inline JamGame/Assets/Scripts/Employee/ExtendedInfo/View.cs:62]: We can move `ExtendedInfo` into Employee and make this dependencies `[Required]` and just turn on/off `ExtendedInfo`

File: JamGame/Assets/Scripts/Employee/ExtendedInfo/View.cs [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+            foreach (Quirk quirk in personality.Quirks)
+            {
+                IResourceLocation view_location = quirkModelViewMap[quirk.Uid];
+
+                _ = Instantiate(
+                    AddressableTools<QuirkView>.LoadAsset(view_location),
+                    quirksContainer
+                );
+            }
```
Comment (Kiuh) [inline JamGame/Assets/Scripts/Employee/ExtendedInfo/View.cs:101]: Move this code into 3 or four other methods.

File: JamGame/Assets/Scripts/Employee/EmployeeImpl.cs [hunk 3]
```diff
@@ -76,6 +79,45 @@ private struct AppliedBuff
         [SerializeField]
         private Vector2 keepDistanceFromNextInLine = new(1.0f, 1.5f);
 
+        private void Awake()
+        {
+            appliedBuffs.CollectionChanged += (s, e) =>
+            {
```
Comment (Kiuh) [inline JamGame/Assets/Scripts/Employee/EmployeeImpl.cs:85]: Make method instead of this lambda and name `s` and `e` `sender` and `event` respectively

**Review by Kiuh — CHANGES_REQUESTED** [review]
image
Correct canvas settings and remove canvas scaler.

File: JamGame/Assets/Scripts/Employee/EmployeeImpl.cs (hunk 3 above)
Comment (mertwole) [reply JamGame/Assets/Scripts/Employee/EmployeeImpl.cs]: `s` and `e` are widely used abbreviations in the context of c# events when the lambda is subscribed to event

**Review by Kiuh — APPROVED** [review]

**Pull request merged**